client/my-sites/site-settings/related-posts/index.jsx

```diff
@@ -40,8 +40,14 @@ const RelatedPosts = ( {
 						text={ translate(
 							'Automatically displays similar content (related posts) at the end of each post.'
 						) }
-						link="https://jetpack.com/support/related-posts/"
+						link="https://en.support.wordpress.com/related-posts/"
 					/>
+					
+					<p className="related-posts__explanation"> 
+						{ translate( 
+							'The following settings will impact all related posts on your site, except for those you created via the block editor:'
```

Comment:
Did we run `prettier` here? This line seems a bit too long.

Reply:
What do you think about:

"These settings won't apply to related posts added using the block editor."

Reply:
Yeah, I feel like that wording is clearer anyway, switched to it.

Reply:
Note that if we change this, we'll need to also change it in Jetpack for consistency (cc @jeherve): https://github.com/Automattic/jetpack/blob/14f5c658d035af51538bdeeb43b6c4601111a541/_inc/client/traffic/related-posts.jsx#L71

Reply:
I've just merged the Jetpack part of this. 👍 	
https://github.com/Automattic/jetpack/pull/11342